tribuo-tutorials/README.md

```diff
@@ -0,0 +1,35 @@
+# Tribuo tutorials
+
+This project shows how to run [Tribuo Machine Learning](https://tribuo.org) algorithms with Conclave.
+
+The code is based on [Tribuo's tutorials](https://tribuo.org/learn/4.0/tutorials/), which cover the following examples:
+* [Classification](https://tribuo.org/learn/4.0/tutorials/irises-tribuo-v4.html)
+* [Clustering](https://tribuo.org/learn/4.0/tutorials/clustering-tribuo-v4.html)
+* [Regression](https://tribuo.org/learn/4.0/tutorials/regression-tribuo-v4.html)
+* [Anomaly Detection](https://tribuo.org/learn/4.0/tutorials/anomaly-tribuo-v4.html)
+* [Configuration](https://tribuo.org/learn/4.0/tutorials/configuration-tribuo-v4.html)
+
+In this sample the client is using Mail to communicate with the enclave.
+Each mail contains a serialized request for the enclave to execute some action, i.e.,
+training models and obtaining test evaluation results.
+
+Kotlin serialization is used to serialize requests and responses to and from the enclave.
+
+To facilitate unit testing each tutorial has been split among several methods which
+are expected to run in a certain order, as each step produces state for the following calls.
+
+The data files have been obtained from the following locations:
```

Comment:
under what license are these data sets made available? Do we need to include a copy of the license? is there anything we need to add to our overall license documentation to say the product now ships with data sets that are covered by license xyz?

Reply:
I've added a section which I hope covers the needs referenced on the respective websites. I haven't found license restrictions yet but included the necessary sources and acknowledgements as requested by the sources.